crawl-ref/source/dat/descript/unrand.txt

```diff
@@ -634,12 +634,12 @@ contamination almost as effectively. Anyone reckless enough to place it on
 their head would find their magic regeneration dramatically increased, at the
 cost of multiplying any magical contamination received.
 %%%%
-# TAG_MAJOR_VERSION == 34
+
```

Comment:
Just removing the TAG_MAJOR_VERSION comment is enough here; the comment shouldn't be replaced with this blank line.